src/meta/app/src/schema/catalog.rs

```diff
@@ -63,7 +63,7 @@ impl CatalogOption {
 /// Option for creating a iceberg catalog
 #[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, Eq)]
 pub struct HiveCatalogOption {
-    pub address: String,
+    pub hive_metastore: String,
```

Comment:
If we change the `hive_metastore` to `metastore_address` it will more general, user not care this option only for hive, we can re-use it if we have another catalog like hive in feature.